packages/block-library/src/footnotes/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { formatListNumbered as icon } from '@wordpress/icons';
+import { registerFormatType } from '@wordpress/rich-text';
+
+/**
+ * Internal dependencies
+ */
+import initBlock from '../utils/init-block';
+import edit from './edit';
+import metadata from './block.json';
+import { formatName, format } from './format';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	icon,
+	edit,
+};
+
+// Would be good to remove the format and HoR if the block is unregistered.
+registerFormatType( formatName, format );
```

Comment:
Should we move this into the `init` function instead? 

This is causing an API request in alternative block editors (see https://github.com/WordPress/gutenberg/issues/53874#issuecomment-2215230258 ) even if the block is not registered.